### ISSUE TITLE
Why was this topic deleted?

### ISSUE THREAD

**Issue by Driadasa** [UnamSanctam/SilentCryptoMiner#2230]
<img width="985" alt="Screenshot_1" src="https://github.com/UnamSanctam/SilentCryptoMiner/assets/86801280/9c09767b-a4e2-4ed3-9939-a5cc99834419">
<img width="878" alt="Screenshot_2" src="https://github.com/UnamSanctam/SilentCryptoMiner/assets/86801280/2f6800cd-97f8-4388-8082-247fcdfb1744">

**Comment by UnamSanctam** [comment]
I would assume since it contains IP addresses, since then they sometimes get removed by GitHub, it happens semi-often, especially when it's an account without much activity that opens the issue. It's also possible that it's only inside containment if someone reported it and will return once checked by GitHub. You can also see that some other issues have been in containment before if there are two issues in the repository, because user opens issue -> goes to containment -> they open a new identical issue -> issue gets released from containment -> there are now two issues.

**Comment by wedobecrazy** [comment]
@UnamSanctam 
still would like to hear an actual response about the nanopool stuff
[nanopool](https://xmr.nanopool.org/account/4Aw8Echp2Hrhc5ussZ5cX1bKS6AFJUqFMJH9373M819NCLMVs4DctwGgtTg1ixc8oqVhZNeKCSTS776xoihXmX8SNYx7vtv)

![2024-02-27_18-37](https://github.com/UnamSanctam/SilentCryptoMiner/assets/161496969/dc021b0f-656c-4e23-96cc-6b00a8ceb98a)

and i see you mentioned this here too...

**Comment by UnamSanctam** [comment]
That's the old example/donation address for the miner, I don't have access to that wallet anymore so I had to create a new one a few years ago, which is why it's different now.

**Comment by wedobecrazy** [comment]
but xmr is still getting mined on [nanopool](https://xmr.nanopool.org/account/4Aw8Echp2Hrhc5ussZ5cX1bKS6AFJUqFMJH9373M819NCLMVs4DctwGgtTg1ixc8oqVhZNeKCSTS776xoihXmX8SNYx7vtv)

**Comment by wedobecrazy** [comment]
and still making withdrawals
![2024-02-27_19-42](https://github.com/UnamSanctam/SilentCryptoMiner/assets/161496969/0d0af67f-c00f-4199-a2ce-fc078fa3fd81)

**Comment by UnamSanctam** [comment]
>but xmr is still getting mined on [nanopool](https://xmr.nanopool.org/account/4Aw8Echp2Hrhc5ussZ5cX1bKS6AFJUqFMJH9373M819NCLMVs4DctwGgtTg1ixc8oqVhZNeKCSTS776xoihXmX8SNYx7vtv)

Yes people can still choose to mine to it for whatever reason, I can't prevent them from doing so, I can't access it though so the XMR is just going out into the void.

>and still making withdrawals

Withdrawals are automatic.

**Comment by msfcon5ol3** [comment]
> > but xmr is still getting mined on [nanopool](https://xmr.nanopool.org/account/4Aw8Echp2Hrhc5ussZ5cX1bKS6AFJUqFMJH9373M819NCLMVs4DctwGgtTg1ixc8oqVhZNeKCSTS776xoihXmX8SNYx7vtv)
> 
> Yes people can still choose to mine to it for whatever reason, I can't prevent them from doing so, I can't access it though so the XMR is just going out into the void.
> 
> > and still making withdrawals
> 
> Withdrawals are automatic.

And making over $2K every month , this is quite a waste of money since no one has access to the private key for that wallet.

**Comment by UnamSanctam** [comment]
>And making over $2K every month , this is quite a waste of money since no one has access to the private key for that wallet.

Yeah sadly it's a waste, and since mining is decentralized it's not like it's possible to contact those that are still mining. And those that are mining must still be on the 1 version of the miner since that's when the address was an example address, which is pretty surprising if that's the case (version 1 was a long time ago). It's also possible that some other person is selling their own modified version of my miner (there are a few) and believe that I still have that address, and thus maybe give donations (hashrate) to it. Personally I think that's more likely than a lot of people using version 1 of the miner.

**Comment by Driadasa** [comment]
> That's the old example/donation address for the miner, I don't have access to that wallet anymore so I had to create a new one a few years ago, which is why it's different now.

You said you don't use donations now, so what are these addresses for?

**Comment by UnamSanctam** [comment]
>You said you don't use donations now, so what are these addresses for?

These are my current donation addresses: https://github.com/UnamSanctam/SilentCryptoMiner#donate. The previous ones were lost when my old hard drive died a few years ago, since then I have many backups of my keys to make sure it can't happen again.

**Comment by Driadasa** [comment]
> > You said you don't use donations now, so what are these addresses for?
> 
> These are my current donation/example addresses: https://github.com/UnamSanctam/SilentCryptoMiner#donate. The previous ones were lost when my old hard drive died a few years ago, since then I have many backups of my keys to make sure it can't happen again.

And why aren't you in the updates cleaning this up?
What's the point of leaving stable mining?

**Comment by UnamSanctam** [comment]
>And why aren't you in the updates cleaning this up?
What's the point of leaving stable mining?

Not sure what you mean, the address hasn't been used since 2020 or 2021.
